.flake8

```diff
@@ -227,3 +267,42 @@ per-file-ignores =
   tests/unit/image_manager/__init__.py: D104
   tests/unit/__init__.py: D104
   share/ansible_navigator/utils/__init__.py: D104
+
+# Count the number of occurrences of each error/warning code and print a report:
+statistics = true
+
+# flake8-eradicate
+# E800:
+eradicate-whitelist-extend = isort:\s+\w+
+
+# flake8-pytest-style
+# PT001:
```

Comment:
We appear to only have one violation of this, is it necessary as global now?

Reply:
I don't know, I just synced the config with known good settings.

Reply:
Can we log an issue to remove it later if it falls outside the scope of this PR so it's not fogotten?

Reply:
These settings are explicit because they differ from the given plugin defaults. I don't see how an issue would help. Do you mean that we need an issue to delete the settings if we decide that we don't need the consistency in the tests to be linted?